Lab2/Assign2/main.cpp

```diff
@@ -0,0 +1,36 @@
+//
+//  main.cpp
+//  Assign2
+//
+//  Created by Monisha Gunalan2 on 2017-03-03.
+//  Copyright © 2017 Monisha Gunalan2. All rights reserved.
+//
+
+#include <iostream>
+using namespace std;
+#include "Student.hpp"
+#include "Teacher.hpp"
+#include "Course.hpp"
+#include <vector>
+
+int main(int argc, const char * argv[]) {
+
+    
+    Student *student1 = new Student("John", "Doe", 20, "123 main street", "ottawa", "613 123 1234");
+    Student *student2 = new Student("James", "Doe", 20, "1 first ave", "ottawa", "613 888 8888");
+    Student *student3 = new Student("Kevin", "Doe", 20, "4 march road", "ottawa", "613 111 1111");
```

Comment:
Here there's no reason to allocate dynamically. You can just declare them on the stack because they are going to be copied into the `Course` anyway.

```
Student student1("John", "Doe", ...);
Student student2(...);
...
```